modules/coo-incident-detection-using.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/incident-detection-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-incident-detection-using_{context}"]
+= Using {coo-full} incident detection
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+* You have installed the {coo-full}.
+* You have installed the {coo-full} monitoring UI plugin with incident detection enabled.
+
+
+.Procedure
+
+. In the Administrator perspective of the web console, click on *Observe* -> *Incidents*.
+
+. The Incidents Timeline UI shows the grouping of alerts into *incidents*. The color coding of the lines in the graph corresponds to the severity of the incident.
```

Comment:
Add a note that once the feature is enabled, it will take at least 10 minutes to process the correlations before you see the timeline.

Reply:
In addition, the analysis and grouping into incidents is performed only for alerts that are firing after feature enablement. Alerts resolved prior to feature enablement are not included.